qutip/core/cy/qobjevo.pyx

```diff
@@ -5,6 +5,7 @@ import numpy as np
 import numbers
 import itertools
 from functools import partial
+from cpython.exc cimport PyErr_CheckSignals
```

Comment:
I don't see PyErr_CheckSignals used anywhere explicitly. Is this import still needed?

Reply:
Sorry, it's not needed.